api/v1/services/index.js

```diff
@@ -2,10 +2,10 @@ module.exports = {
 	AuthService: require('./AuthService'),
 	EcosystemService: require('./EcosystemService'),
 	MailService: require('./MailService'),
+	RegistrationService: require('./RegistrationService'),
```

Comment:
Oh yeah in that case yes it should lol. I'll change that when I add the other fixes.